contract/contracts/Statements.sol

```diff
@@ -0,0 +1,32 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract Statements {
+
+    struct Request {
+        uint256 from;
```

Comment:
nit: maybe for simplicity - just a calendar year?

Reply:
what is the problem with arbitrary `from` `to`?

Reply:
nothing wrong, just thoughts that probably easier for client to set the input + naturally we could just collect fees per year (per request). not important atm